### ISSUE TITLE
Date of original file in Change File Date and dateadded in nfo

### ISSUE THREAD

**Issue by andrsm** [Radarr/Radarr#4913]
Hi
Would it be possible to add a new option for "Date of original file" to "Change File Date" in File Management options in v3?
So if I download a new version of a movie the new file would keep the original file creation date.

In addtion could it be possible to add a dateadded field in the metadata NFO for kodi/emby in v3?
dateadded would then refer to the first time a file was downloaded for this movie (not added to radarr as that might be months before a physical release).

This would be for keeping date added in kodi or emby consistent and not updating for every new version of the movie.

[AB#832](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/832)

**Comment by austinwbest** [comment]
I know, it happens to me as well and is annoying for sure, but unfortunately doesn't change my mind on the fact that a file date is file specific. 

The name of the file also changes which I would imagine plays a factor in that as well?

**Comment by andrsm** [comment]
Yes, I keep the original file name on files. I find it best to have as much information about the file as possible in the filename.

I'm not too fond on the idea of changing the file date, but I'll accept it if it fixes this problem. And since radarr already have an option to alter it (to cinema date or physical date), I thought maybe it could be an option to have a "custom date" (first file creation date).

**Comment by SugarDroid** [comment]
> Can't say that I agree with this. If you have a CAM copy of a movie and it upgrades to the blu-ray it will keep the date from the cam? That is completely wrong.

This simple depends on anyone's point of view. I for sure would rather keep -even if it was a CAM- the date when the movie first appeared into the library, than let it annoys the hell out of me by keep popping up again... and again... and again... and again... and... -whenever a new version gets downloaded- as a new movie added to the library today. For me that is what would be completely wrong.

> I know, it happens to me as well and is annoying for sure, but unfortunately doesn't change my mind on the fact that a file date is file specific.

This again just depends on point of view. I'm looking at this in a way that a file's date is movie specific and keep the same date even if the file itself is changed. This is why, I also use this feature to prevent this to happening, while -seems to me- you rather let it annoys you.

**Comment by solidsnakex37** [comment]
This would be such a quality of life feature if it could be implemented, even as an advanced option hidden in the settings. I don't know how time consuming it would be to add this, all I imagine is a check box to "Ignore Date-Added when importing upgrades" option so that as movies go through the many upgrades, it will note overwrite that tag in the .nfo file. 

This would add the following quality of life changes: 

Movies will only appear in the "Recently Added" section once
Movies will retain their watched status even if an upgrade takes place (please)
Sorting by "Date Added" will be accurate and true, as libraries will sort movies by the date first added, not by upgrade date.

**Comment by dRuEFFECT** [comment]
+1, I support this, please consider

**Comment by quadcom** [comment]
Can I +10 this?

Using Kodi, and if I lose my DB and have to rescan, all the items have the date added as the new date they were imported. Not the previous date I actually added it to my collection. Having this 'imported' date in the nfo could be use by Kodi to reimport and still keep the library as it was before the crash.

**Comment by shunopoli** [comment]
I know this is a year old, but would love this feature

**Comment by Sectimus** [comment]
I would absolutely love to have this one in. I get new media all the time that my family would like to be notified about when the log in to the service, but they can never see what is actually _**new**_. Since the "_Recently added_" categories are derived from the media file creation date, so it makes sense for me to be able to control that date to just be the date of first import. If radarr see's that there is a file that is already there, then can't we just copy the date of that file to overwrite our new version?

![Image](https://github.com/user-attachments/assets/c8bcc14e-e322-4619-abde-06aa913cd27a)

Not asking for defaults here even, but the option would be nice. As right now there is no automated control of this.

**Comment by dedors** [comment]
I'm also waiting for years for new options for "Change File Date". My preference would be on imported/upgraded. When I download old movies, I don't want them to be lost in the void. 
The perfect balance would be newly imported, and upgraded where I changed the quality profile, started monitoring again or manually initiated a new search, but not auto upgraded, but that implementation may be harder to do.
